include/xeus-python/xdebugger.hpp

```diff
@@ -71,6 +71,7 @@ namespace xpyt
         std::string m_debugpy_port;
         nl::json m_debugger_config;
         py::object m_pydebugger;
+        std::string m_debugpy_version;
```

Comment:
I think storing a boolean stating if the `copy_to_globals_request` is available might be more efficient.

Reply:
OK, I thought we could keep it generic for future changes.